nilearn/image/resampling.py

```diff
@@ -300,8 +297,9 @@ def _resample_one_img(
 
     # Suppresses warnings in https://github.com/nilearn/nilearn/issues/1363
     with warnings.catch_warnings():
-        if _compare_version(scipy.__version__, ">=", "0.18"):
-            warnings.simplefilter("ignore", UserWarning)
+        warnings.filterwarnings(
+            "ignore", message=".*has changed in SciPy 0.18.*"
+        )
```

Comment:
figured that it'd be more specific rather than ignoring all user warnings, WDYT?